integration-testing/src/functionalTest/kotlin/kotlinx.atomicfu.gradle.plugin.test/cases/MppProjectTest.kt

```diff
@@ -31,28 +31,28 @@ class MppProjectTest {
     @Test
     fun testMppWithEnabledJsIrTransformation() {
         mppSample.enableJsIrTransformation = true
-        assertTrue(mppSample.cleanAndBuild().isSuccessful)
```

Comment:
These are just minor changes of the error output format. I could've considered adding them separately tho.